src/routes/mod/[modId]/+page.svelte

```diff
@@ -101,7 +101,9 @@
       <h1 class="text-4xl font-bold">{$mod.data.mod.name}</h1>
       <div>
         {#if canUserEdit}
-          <button class="variant-ghost-primary btn" on:click={() => goto(base + '/mod/' + modId + '/edit')}>
+          <button
+            class="variant-ghost-primary btn"
+            on:click={() => goto(base + '/mod/' + modId + '/edit', { invalidateAll: true })}>
```

Comment:
`invalidateAll` is not really a great tool, you almost never need to clear the entire cached data. If you want to forcefully fetch the latest data, urql has `requestPolicy: 'network-only'`

Reply:
I saw that, but didn't see how to use that, I'll take another look on how to make it use that instead as this fixed a lot stale data issues that made tags mess up.